ext/javy/javy.rb

```diff
@@ -45,7 +46,8 @@ def target
 
     def ensure_installed
       delete_outdated_installations
-      install unless Installer.installed?(target: target)
+      install.unwrap { |e| raise e } unless Installer.installed?(target: target)
+      true
```

Comment:
I don't think that's needed?

Reply:
I moved the `ensure_installed` call into the result chain on line 28 https://github.com/Shopify/shopify-cli/pull/1801/files#diff-af8b8f1c7b7f4e64d9c8b7a34b35c47497191d276dfa634bc49311ddc0fffc47R26 as its an important prereq to running `exec`, and this method always produces a Result::Failure unless it returns a non-nil object 🤷

Reply:
You're right, today I learned. 👍